specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -393,14 +536,14 @@
                     { "value": "SgxEnclave", "description": "Intel Software Guard eXtensions" },
                     { "value": "OpenEnclave", "description": "OpenEnclave extensions to SGX" },
                     { "value": "CyResComponent", "description": "IoT Edge validation" },
-                    { "value": "AzureGuest", "description": "Azure Guest Attestation" }
+                    { "value": "VSMEnclave", "description": "VSM Enclave Attestation" }
```

Comment:
Does it mean "AzureGuest" cannot be returned by the service anymore?

Reply:
Yes, AzureGuest is no longer supported.  And VSMEnclave is now supported.